apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -40,19 +40,22 @@ marionette('Dialer > Keypad', function() {
     keypadTap(three);
   }
 
-  function keypadTap(elem, longPress) {
-    var number = subject.client.findElement(selectors.phoneNumber);
-    var length = number.getAttribute('value').length;
+  function typeLongNumber() {
+    typeNumber();
+    typeNumber();
+    typeNumber();
+    typeNumber();
+    typeNumber();
+  }
 
+  function keypadTap(elem, longPress) {
     if (longPress) {
       actions.longPress(elem, 1).perform();
     } else {
       actions.tap(elem).perform();
     }
 
-    client.waitFor(function() {
-      return (number.getAttribute('value').length == (length + 1));
-    });
+    subject.client.helper.wait(200);
```

Comment:
Why did you change keypadTap() to wait for an explicit amount of time instead of the previous condition? Explicit waits in tests are prone to intermittent failures so you'll have to change this.